care/facility/models/patient.py

```diff
@@ -0,0 +1,161 @@
+from django.db import models
+from fernet_fields import EncryptedCharField, EncryptedTextField
+from partial_index import PQ, PartialIndex
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'partial_index'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5331745)